graphics/CViewport.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+#include "rage/grcViewport.hpp"
+
+struct CViewportGame
+{
+public:
+    virtual ~CViewportGame() = 0;
+
+private:
+    char m_pad[8];
+
+public:
+    grcViewport viewport;
```

Comment:
`CViewportGame isn't part of the `rage` namespace, how is this able to compile? I'm guessing the headers haven't been added to the cpp test compile file...